content/en/docs/setup/getting-started/index.md

```diff
@@ -88,6 +88,21 @@ Follow these steps to get started with Istio:
     ✔ Installation complete
     {{< /text >}}
 
+    {{< warning >}}
```

Comment:
I think this is the wrong place for this content. In the change note for this set of changes, I recommended:

# docs is a list of related docs to the change.
docs:
- docs/setup/platform-setup/openshift/
- docs/setup/install/istioctl/#install-a-different-profile

and I just noticed this PR was submitted:

https://github.com/istio/istio.io/pull/8334#pullrequestreview-514844772

Reply:
I agree this is the wrong place for this. How about just a simple warning after line 80, something like this:
```

{{< warning >}}
If your platform has a custom demonstration profile, e.g., Openshift, use it in the following command,
instead of the `demo` profile. Refer to your [platform instructions](/docs/setup/platform-setup/) for details.
{{< /warning >}}
```
And then show the command in docs/setup/platform-setup/openshift/

Reply:
@frankbu I think this might be a good compromise so that users are aware of using the profile that suits their platform. 

Updated.

Reply:
@sdake I will add the content to the docs you mentioned.